python/runtime/dbapi/connection.py

```diff
@@ -102,6 +102,11 @@ def __init__(self, conn_uri):
             if len(l) == 1:
                 self.params[k] = l[0]
 
+    def param(self, param_name, default_value=""):
```

Comment:
Seems that this method is not used?

Reply:
This is a tool api, we will need it in our submitter code.